LEAF_Request_Portal/form.php

```diff
@@ -3061,6 +3066,23 @@ public function query($inQuery)
             }
         }
 
+        if ($joinRecordResolutionBy === true) {
+            require_once 'VAMC_Directory.php';
+            $dir = new VAMC_Directory;
+
+            $res2 = $this->db->prepared_query('SELECT recordID, userID as resolvedBy, stepID, time FROM action_history
+                                                LEFT JOIN workflow_routes USING (stepID)
+                                                WHERE recordID IN (' . $recordIDs . ')
+                                                and nextStepID = 0
+                                                ORDER BY time', array());
+
+            foreach ($res2 as $item) {
+                $user = $dir->lookupLogin($item['resolvedBy']);
+                $nameResolved = isset($user[0]) ? "{$user[0]['Lname']}, {$user[0]['Fname']}" : $item['resolvedBy'];
```

Comment:
3081 in form.php ternary looks like it's assigning not checking value equality

Reply:
? it is assigning the first and last name if it can find it. What were you expecting that isn't there?

Reply:
the inline evaluates if/then, not a = b, and is correct usage.